README.md

```diff
@@ -131,10 +131,24 @@ Cleanup steps:
 3. Delete the persistent volume directories
     * __~/.local/share/osbs-box/pv/*__ by default
     * They tend to be owned by other users, you may need `sudo`
-5. _OPTIONAL_: Delete all the remaining data
+4. _OPTIONAL_: Delete all the remaining data
     * __~/.local/share/osbs-box/__ by default
     * You should never _need_ to do this - unless you are breaking up with OSBS-Box :cry:
 
+The __cleanup.yaml__ playbook runs some of these steps for you based on what `--tags` you specify:
```

Comment:
It would be better to inform users that playbook exists before listing manual steps

Reply:
Good point, in fact there no longer seems to be much of a reason to list the manual steps at all